packages/create/bin/create-stacks-workspace.ts

```diff
@@ -134,9 +147,49 @@ async function determineAppName(
         });
 }
 
+// eslint-disable-next-line unicorn/prevent-abbreviations
+async function determineE2eTestRunner(
+    parsedArguments: yargs.Arguments<CreateStacksArguments>,
+) {
+    if (!(parsedArguments.e2eTestRunner || parsedArguments.interactive)) {
+        return Promise.resolve(E2eTestRunner.None);
+    }
```

Comment:
I assume this will mean that when running via the StacksCLI, as we won't be passing an e2e argument, this will default the test runner to none?

Reply:
correct